src/components/UI/Text/Textarea.vue

```diff
@@ -36,26 +36,30 @@
       >
         <inline-svg :src="startIconObj"></inline-svg>
       </div>
-
       <!-- input text area -->
-      <textarea
-        class="p-2 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
+
+      <div
+        contenteditable="true"
+        class="pt-7 h-30 textbox pl-0 border placeholder-blueGray-300 text-blueGray-600 bg-white disabled:bg-gray-200 rounded text-md border-blueGray-300 focus:outline-none focus:ring focus:border-transparent focus:shadow-outline w-full border-gray-200 disabled:cursor-not-allowed"
```

Comment:
This has become a long enough class :p
I think you should put it in a variable and then link the variable here.
For reference, see the rest of code where we've added tailwind classes as variables and used in HTML

Reply:
sounds good! will make these changes